content/200-concepts/100-components/03-prisma-migrate/010-mental-model.mdx

```diff
@@ -0,0 +1,163 @@
+metaTitle: 'A mental model for Prisma Migrate'
+metaDescription: 'A mental model guide for working with Prisma Migrate in your project'
+tocDepth: 3
+---
+
+<TopBlock>
+
+This guide provides a conceptual overview of database migrations using Prisma Migrate when working with relational databases. It covers: what database migrations are, their value, and what Prisma Migrate is and how you can evolve your database schema with Prisma Migrate in different environments.
+
+**Does not apply to MongoDB** because of its _schemaless_ nature
+
+</TopBlock>
+
+## What are database migrations?
+
+Database migrations are a controlled set of changes that modify and evolve the structure of your database schema. Migrations help you transition your database schema from one state to another. For example, within a migration you can create or remove tables and columns, split fields in a table, or add types and constraints to your database.
+
+## Evolve your database schema
+
+There are two main patterns you can use to evolve your database schema:
+
+- **Database-first migration:** with this pattern, you define the structure of your database and apply it to your database using SQL. You then _introspect_ the database to generate the code that describes the structure of your database to sync your application and database schema.
+
+![](https://user-images.githubusercontent.com/33921841/194378557-a18d5ad3-96a8-45da-8377-e078893ef2bb.png)
+
+- **Model/Entity-first migration:** with this pattern, you define the structure of the database schema with code and then use a migration tool to generate the SQL, for example, for syncing your application and database schema.
+
+![](https://user-images.githubusercontent.com/33921841/194377029-305dc60b-f651-46e4-87f1-508884ebd972.png)
+
+The migration files (SQL) should ideally be stored together with your application code. They should also be tracked in version control and shared with the rest of the team working on the application.
+
+Migrations provide _state management_ which helps you to track the state of the database.
+
+Migrations also allow you to replicate the state of a database at a specific point in time which is useful when collaborating with other members of the team, e.g. switching between different branches.
+
+For further information on database migrations, see the [Prisma Data Guide](https://www.prisma.io/dataguide/types/relational/what-are-database-migrations).
+
+## What is Prisma Migrate?
+
+Prisma Migrate is a database migration tool that supports _model/ entity-first_ migration and _database first_ migration patterns to manage database schemas in your local environment and in production.
+
+The workflow when using Prisma Migrate in your project would be iterative and look like this:
+
+- Feature branch (locally)
+  1. Define your Prisma schema – the source of truth
+  1. Use one of the [Prisma CLI](/reference/api-reference/command-reference) commands to sync your Prisma schema with the database schema
```

Comment:
I think it would be worth mentioning here what kind of DB this is

```suggestion
  1. Use one of the [Prisma CLI](/reference/api-reference/command-reference) commands to sync your Prisma schema with the database schema of your local development database
```